### DISCUSSION TITLE
Indentation issue in /dev/shm/go2rtc.yaml

### DISCUSSION THREAD

**Discussion by my-umd** [blakeblackshear/frigate#6203] [General]
Not sure if this has been reported.

In 0.12.0, when checking the generated /dev/shm/go2rtc.yaml, I see the indentation might not set correctly.  E.g., from my go2rtc dashboard > Config:
```
streams:
  backyard:
  - rtsp://xxx:xxx@192.168.x.x/live
  - ffmpeg:backyard#audio=aac
```

There should be more indentation for the two lines start with '-' (rtsp and ffmpeg).

Not sure if this is a bug, and where to submit bug report.  The go2rtc dashboard seems working.  So this probably does not affect the functionality.  Maybe just for cosmetic.

**Comment by NickM-27** [comment]
That yaml is directly generated from the python yaml package

**Reply by my-umd** [reply]
I see.  Thanks @NickM-27.  Just wanted to report things that do not look right.
FYI: I double checked the example on https://jsonformatter.org/yaml-formatter.  This online tool also shows that the two lines need to be further indented.  Maybe both are acceptable.

**Comment by AlexxIT** [comment]
There is no requirements for count of this indents

**Reply by my-umd** [reply]
Yes, thanks @AlexxIT.  I also tested it with the yaml package.  Both worked fine.
